src/components/activities/adminActivity/types.ts

```diff
@@ -0,0 +1,18 @@
+import { TUserCredentials } from "../../../state/main/types";
+import { IApiResponse } from "../../../state/types";
+import { PatientDTO } from "../../../generated";
+
+export type TActivityTransitionState = "IDLE" | "TO_PATIENT_EDITING";
+
+export type IAdminSection =
+  | "wards"
+  | "diseases"
+  | "exams"
+  | "operations"
+  | "vaccines"
+  | "priceList"
+  | "suppliers"
+  | "telemetry"
+  | "usersGroups"
```

Comment:
nitpicking and opinionated

I would recommend using only the "users" keyword as prop value and url element because `usersGroups` might be interpreted as "administrate users' groups", not "administrate users and groups".

(again, this is totally opinionated, feel free to disregard this comment 😊)